common/components/reduction.hpp.inc

```diff
@@ -109,7 +109,6 @@ __device__ void reduce(const Group &__restrict__ group,
 {
     const auto local_id = group.thread_rank();
 
-#pragma unroll
     for (int k = group.size() / 2; k >= config::warp_size; k /= 2) {
```

Comment:
This group is larger than warp_size, so we can not unroll this loop in the compile time.

Reply:
Makes sense, thanks for clearing that up.